src/vs/code/electron-sandbox/workbench/workbench.js

```diff
@@ -112,5 +114,96 @@
 		return window.MonacoBootstrapWindow;
 	}
 
+	/**
+	 * @param {INativeWindowConfiguration & NativeParsedArgs} configuration
+	 */
+	function showPartsSplash(configuration) {
```

Comment:
This is a 100% copy of the other one in `electron-browser`. Once we have enabled sandbox, the duplication will go away.